content/guides/identity-management/quick-start/2-configure-okta.md

```diff
@@ -0,0 +1,262 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Configure Okta
+
+Our next step in the Okta / Box integration is to create and configure the Okta
+application and users, then extract a few pieces of information that we will
+need to connect to Okta in our application.
+
+<ImageFrame noborder center shadow>
+  ![Okta Application Dashboard](./img/okta-applications.png)
+</ImageFrame>
+
+For this tutorial we will be starting with a blank Okta application and user
+dashboard to avoid any negative effects on existing installations that may be
+in place, and to ensure that we have admin rights to the instance.
+
+## Create an Okta Application
+
+Starting from the [Okta developer site][okta-dev], sign up for a new developer
+account, or log in under your personal account if you already have one.
+
+After logging in, you should see the Okta dashboard. Click on the **Admin**
```

Comment:
I created a Okta free developer account and actually ended up on the developer dashboard directly. E.g. https://dev-410133-admin.okta.com/admin/getting-started

Reply:
Added adjusted language.